### PR TITLE
Add addresses for Fault Proof contracts on Mainnet L1

### PR DESCRIPTION
**Pull request by cbfyi**
**What changed? Why?**

Similar to https://github.com/base-org/web/pull/656, but for mainnet.

**Notes to reviewers**

Addresses can be verified using https://github.com/ethereum-optimism/superchain-ops/pull/352/files#diff-0aee6ef7ca0ce5497080deb4551d5f285d7747b8aabd63c250d5b356fd2f8225

Or alternatively, calling `0x43edB88C4B80fDD2AdFF2412A7BebF9dF42cB40e` (see this [PR](https://github.com/base-org/web/pull/1167) for details).

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/base-docs/docs/building-with-base/base-contracts.md [hunk 1]
```diff
@@ -73,12 +73,19 @@ hide_table_of_contents: true
 | Name                         | Address                                                                                                               |
 | :--------------------------- | :-------------------------------------------------------------------------------------------------------------------- |
 | AddressManager               | [0x8EfB6B5c4767B09Dc9AA6Af4eAA89F749522BaE2](https://etherscan.io/address/0x8EfB6B5c4767B09Dc9AA6Af4eAA89F749522BaE2) |
+| AnchorStateRegistryProxy     | [0xdB9091e48B1C42992A1213e6916184f9eBDbfEDf](https://etherscan.io/address/0xdB9091e48B1C42992A1213e6916184f9eBDbfEDf) |
+| DelayedWETHProxy             | [0xa2f2aC6F5aF72e494A227d79Db20473Cf7A1FFE8](https://etherscan.io/address/0xa2f2aC6F5aF72e494A227d79Db20473Cf7A1FFE8) |
```
Comment (awilliams1-cb) [inline apps/base-docs/docs/building-with-base/base-contracts.md:77]: Similar comment as the one in [PR1167](https://github.com/base-org/web/pull/1167#discussion_r1819654357)
```suggestion
| DelayedWETHProxy (FDG)       | [0xa2f2aC6F5aF72e494A227d79Db20473Cf7A1FFE8](https://etherscan.io/address/0xa2f2aC6F5aF72e494A227d79Db20473Cf7A1FFE8) |
| DelayedWETHProxy (PDG)       | [0x3E8a0B63f57e975c268d610ece93da5f78c01321](https://etherscan.io/address/0x3E8a0B63f57e975c268d610ece93da5f78c01321) |
```

**Review by awilliams1-cb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for awilliams1-cb @ 2024-10-28 22:04:40 UTC
User must have write permissions to review

**Review by henridevieux — APPROVED** [review]
Addresses are accurate as compared with my local notes from terraform output of contract creation

**Review by wbnns — APPROVED** [review]

**Pull request merged**